less/inc/single.less

```diff
@@ -1,474 +1,464 @@
 
 /*------------------------------------------------------------------------------
-  Single - Styling for single posts & pages.
+Single - Styling for single posts & pages.
 ------------------------------------------------------------------------------*/
 
 
 /*------------------------------------------------------------------------------
-    1.0 - Normal layout overrides (override two column display)
+ 1.0 - Normal layout overrides (override two column display)
 ------------------------------------------------------------------------------*/
 
 body.normal.single-post,
 body.normal.page {
 
-  /* 1.1 - Header */
-
-  article.post > header,
-  article.page > header {
-
-    clear: both;
-    margin: 24px 17.0213%;
-    position: relative;
-
-    // top tag
-    h5.top-tag {
-      position: absolute;
-      left: 0;
-    }
-
-    h1 {
-      padding-top: 26px;
-    }
-
-    // hide normal date display.
-    .sep {
-      display: none;
-    }
-
-    // edit link
-    .edit-link {
-      padding-left: 5px;
-    }
-
-  }
-
-  /* 1.2 - Hero */
-  .hero {
-    p.wp-caption-text,
-    p.wp-media-credit {
-      margin:0 17.0213% 12px;
-    }
-  }
-
-  /* 1.3 - Sidebar (left of post) */
-
-  .single #main #sidebar {
-      /* all to undo bootstrap */
-      float: none;
-      width: auto;
-      min-height: 0;
-      margin-left: 0;
-    .widget {
-      /* float widgets */
-      float: left; 
-      clear: left;
-      padding-right: 20px; // add padding for gutter between body text & sidebar
-      width: 17.021276594% - @fluidGridGutterWidth;
-    }
-    .widget:first-child {
-      padding-top: 0; // first widget flush with top of article
-    }
-  }
-
-  /* 1.4 - Entry Content (the main post content) */
-
-  .entry-content {
-    padding: 0 17.021276594%;
-  }
-
-
-  p.intro,
-  p.wp-caption-text {
-    font-style: normal;
-  }
-
-  /* 1.5 - Article Bottom (after post content) */
-
-  .article-bottom {
-    margin: 0 17.021276594%;
-    .post-social {
-      border-top: none;
-      border-bottom: none;
-    }
-  }
-
-  #comments,
-  #disqus_thread {
-    margin: 0 17.021276594%;
-  }
-
-  /* 1.6 - Overrides */
-
-  @media @tabletDown {
-  .single #main #sidebar {
-      display: none;
-    }
-    .article-bottom,
-    #comments,
-    #disqus_thread {
-      margin: 0 @fluidGridColumnWidth;
-    }
-    article.post > header {
-      margin: 24px @fluidGridColumnWidth;
-    }
-    .entry-content {
-      padding: 0 @fluidGridColumnWidth;
-    }
-    .hero {
-      p.wp-caption-text,
-      p.wp-media-credit {
-        margin:0 @fluidGridGutterWidth 12px;
-      }
-    }
-  }
-
-  @media @mobile {
-    .article-bottom,
-    #comments,
-    #disqus_thread {
-      margin: 0;
-    }
-    article.post > header {
-      margin: 24px 0;
-    }
-    .entry-content {
-      padding: 0;
-    }
-  }
+	/* 1.1 - Header */
+
+	article.post > header,
```

Comment:
looks like all of the indentation changed here, we should stick to two spaces in CSS (instead of 4)